algorithms/msf.md

```
@@ -0,0 +1,177 @@
+       (d)-[:ROAD {distance: 6}]->(e)
+
+// Find minimum spanning forest using distance weights
+CALL algo.MSF({
+    relationshipWeightProperty: 'distance'
+}) YIELD src, dest, weight
+RETURN src.name AS from, dest.name AS to, weight AS distance
+ORDER BY weight
+```
+
+**Result:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)